Add link to guide for removing video's audio in RC (#5773)
ppy/osu-wiki: wiki/Ranking_Criteria/en.md

  - **Maximum file size:** 2.5MB
- **A video's dimensions must not exceed a width of 1280 and a height of 720 pixels.** Additionally, upscaling lower resolution video to a higher resolution should be avoided. This ensures video files do not become excessively large or resource intensive.
- **A video's [offset](/wiki/Beatmapping/Offset) must be correct if it synchronizes with the song.** An incorrect offset can result in a misleading visual representation of the song. If the same video appears in multiple difficulties, it must always have the same offset(s).

> - **A video's audio track must be removed from the video file.** The audio track in video is not used in osu!, so removing it reduces that file size of the beatmap. This includes videos with muted audio tracks. A guide to removing the audio can be found [here](/wiki/guides/Compressing_files/en.md).

Reviewer: ```suggestion
- **A video's audio track must be removed from the video file.** The audio track in video is not used in osu!, so removing it reduces that file size of the beatmap. This includes videos with muted audio tracks. *Note: A guide to removing the audio can be found [here](/wiki/Guides/Compressing_files).*
```

> - **A video's audio track must be removed from the video file.** The audio track in video is not used in osu!, so removing it reduces that file size of the beatmap. This includes videos with muted audio tracks. *Note: A guide to removing the audio can be found [here](/wiki/Guides/Compressing_files).*

Reviewer: ```suggestion
- **A video's audio track must be removed from the video file.** The audio track in video is not used in osu!, so removing it reduces that file size of the beatmap. This includes videos with muted audio tracks.
  - *Note: A guide to removing the audio can be found [here](/wiki/Guides/Compressing_files).*
```
checked the file and notes like this one are placed in a subbullet actually

so i guess we can go with something like this

Reviewer 2: ```suggestion
- **A video's audio track must be removed from the video file.** The audio track in video is not used in osu!, so removing it reduces that file size of the beatmap. This includes videos with muted audio tracks.
  - *See [Compressing files § Video](/wiki/Guides/Compressing_files#video) for guidance.*
```
or similar, to get rid of no-info "here" link
